src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="https://fonts.google.com/specimen/Roboto">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
```

Comment:
please follow [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) - when it comes to BEM block  - we want them to as simple as possible